_examples/complex.env

```diff
@@ -0,0 +1,55 @@
+# Environment Variables
+
+
+## ComplexConfig
+## ComplexConfig is an example configuration structure.
+## It contains a few fields with different types of tags.
+## It is trying to cover all the possible cases.
+#
+## Secret is a secret value that is read from a file.
+## (from-file)
+# SECRET=""
+## Password is a password that is read from a file.
+## (from-file, default: '/tmp/password')
+# PASSWORD=/tmp/password
+## Certificate is a certificate that is read from a file.
+## (expand, from-file, default: '${CERTIFICATE_FILE}')
+# CERTIFICATE=${CERTIFICATE_FILE}
+## Key is a secret key.
+## (required)
+# SECRET_KEY=""
+## SecretVal is a secret value.
+## (required, non-empty)
+# SECRET_VAL=""
+## Hosts is a list of hosts.
+## (separated by ':', required)
+# HOSTS=""
+## Words is just a list of words.
+## (comma-separated, from-file, default: 'one,two,three')
+# WORDS=one,two,three
+## Just a comment.
+## (required, default: 'This is a comment.')
+# COMMENT=This is a comment.
+
+## Anon is an anonymous structure.
+#
+## User is a user name.
+## (required)
+# ANON_USER=""
+## Pass is a password.
+## (required)
+# ANON_PASS=""
+
+## NextConfig
+#
+## Mount is a mount point.
+## (required)
+# MOUNT=""
+
+## FieldNames
+## FieldNames uses field names as env names.
+#
+## Quux is a field with a tag.
+# QUUX=""
+
+^ are we missing values?
```

Comment:
@g4s8 I manually edited this file to be what I think a good format would be.
The rules I was going for was:
* `##` prefixes any docs line
* `#` prefixes any field line
* if there is a default, specify that in the field
* a blank `#` between the struct docs and the first field
* a blank line between structs if there are more then one
* the env properties are on a separate line in parenthesis, mostly to keep line length down.

I did notice that I think the FieldNames struct is missing some values, but this is true in the other formats as well, so it seems unrelated to my dotenv work.

Reply:
Looks good to me, thanks so much

Reply:
About missed values from `FieldName` struct: https://github.com/g4s8/envdoc/issues/11